packages/fast-components-react-msft/assets/carousel-hero-content.tsx

```diff
@@ -1,18 +1,16 @@
-import React from "react";
+import React, { Fragment } from "react";
```

Comment:
FYI - You're already bringing in React, you *could* reference here as `React.Fragment` without the import.